hydra-node/json-schemas/api.yaml

```diff
@@ -1599,6 +1694,67 @@ components:
                 }
             }
         }
+    
+    UTxOWithWitness:
+      type: object
+      propertyNames:
+        pattern: "^[0-9a-f]{64}#[0-9]+$"
+      items:
+        $ref: "#/components/schemas/TxOutWithWitness"
+      example:
+        {
+            "09d34606abdcd0b10ebc89307cbfa0b469f9144194137b45b7a04b273961add8#687": {
+                "address": "addr1w9htvds89a78ex2uls5y969ttry9s3k9etww0staxzndwlgmzuul5",
+                "value": {
+                    "lovelace": 7620669
+                },
+                "witness": {
+                  "datum": "02",
+                  "plutusV2Script": {
+                      "cborHex": "420606",
+                      "description": "",
+                      "type": "PlutusScriptV2"
+                  },
+                  "redeemer": "21"
+                }
+            }
+        }
+
+    ScriptInfo:
+      description: |
+        Script data needed to spend user provided utxo
+      required:
+      - redeemer
+      - datum
+      - plutusV2Script
+      payload:
+        type: object
+        properties:
+          redeemer:
+            type: string
+          datum:
+            type: string
```

Comment:
Should: use `$ref: "#/components/schemas/Cbor"` to be more precise what kind of string is used for `redeemer` and `datum`